src/main/java/our/member/domain/Token.java

```diff
@@ -0,0 +1,28 @@
+package our.member.domain;
+
+import our.member.error.ExpiredTokenException;
+import our.member.error.InvalidTokenException;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+public class Token {
+    private Long id;
+    private TokenVo tokenVo;
+    private UserEntity userEntity;
+    private TokenState tokenState;
+    private LocalDateTime expiresAt;
+
+    public Token(UUID id, String tokenVo, UserEntity userEntity, TokenState tokenState, LocalDateTime expiresAt) {
+        if (!tokenState.equals(TokenState.VALID)) {
+            throw new InvalidTokenException();
+        }
+        if (expiresAt.isBefore(LocalDateTime.now())) {
```

Comment:
아래와 같이 static import로 진행하는건 어떨까요? ☺️

```suggestion
        if (expiresAt.isBefore(now())) {
```